package.json

```diff
@@ -13,7 +13,9 @@
     "speculate": "speculate",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test:e2e": "npx cypress run",
-    "test:e2e:interactive": "npx cypress open"
+    "test:e2e:interactive": "npx cypress open",
+    "test:storybook": "CYPRESS_baseUrl=http://localhost:9001 npm run test:e2e -- --spec cypress/integration/storybook_spec.js",
+    "test:storybook:interactive": "CYPRESS_baseUrl=http://localhost:9001 npm run test:e2e:interactive"
```

Comment:
@jamesbrumpton Why does this line not require the `-- --spec cypress/integration/storybook_spec.js` on the end?

It might be correct, I'd just like to know why it's different.

Reply:
That's a question for @pjlee11

Reply:
Regardless I cannot think of a reasonable use case for `test:storybook:interactive` we were only planning on having `test:storybook` to ensure we don't break it accidentally, not as the start of testing every component within it. Suggest we remove the line.